admin/index.php

```diff
@@ -326,22 +333,27 @@ function bounceOut(){ header("Location: http://davesmapper.com/admin"); exit; }
 									<div class="fieldset">
 										<h4 class="legend">Info</h4>
 										<input type="text" name="alink" id="alink" autocomplete="off" />
-										<input type="text" name="aemail" id="aemail" autocomplete="off" />
 										<input type="text" name="aicon" id="aicon" class="little" autocomplete="off" /><br/>
 											<label for="alink" class="labelTxt tBox">Link URL</label>
-											<label for="aemail" class="labelTxt tBox">Email</label>
 											<label for="aicon" class="labelTxt nBox">.png</label>
 									</div>
 									<div class="fieldset">
+										<h4 class="legend">Auth</h4>
+										<input type="text" name="aemail" id="aemail" autocomplete="off" />
```

Comment:
This is a form for adding a new user, so I wouldn't be autocompleting with my own email address. Once I build in the part for users to create their own accounts, that form will not have autocomplete turned off.